models/brats_mri_axial_slices_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,171 @@
+# Model Overview
+A pre-trained model for 2D Latent Diffusion Generative Model on axial slices of BraTS MRI.
+
+This model is trained on BraTS 2016 and 2017 data from [Medical Decathlon](http://medicaldecathlon.com/), using the Latent diffusion model (Rombach, Robin, et al. "High-resolution image synthesis with latent diffusion models." Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition. 2022. https://openaccess.thecvf.com/content/CVPR2022/papers/Rombach_High-Resolution_Image_Synthesis_With_Latent_Diffusion_Models_CVPR_2022_paper.pdf).
```

Comment:
could you add a references section, and modify this place?
You can refer to:
https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_generative_diffusion#references

Reply:
Done. Thanks!